q2_qemistree/_hierarchy.py

```diff
@@ -72,46 +101,50 @@ def make_hierarchy(csi_results: CSIDirFmt,
         one or more CSI:FingerID output folder
     feature_table : biom.Table
         one or more feature tables with mass-spec feature intensity per sample
-    qc_properties : bool, default True
+    feature_data : pd.DataFrame
+        metadata (row ID, row m/z) about features in the feature table
+    mz_tolerance : float
+        maximum allowable tolerance in m/z of parent ions
```

Comment:
Can you add a description for why this is 0.01? Referring to our conversation with Kai should be fine, we can later write down why this value is suggested.

Reply:
Also worth acknowledging that if we increase the tolerance we'll merge more things.